src/j1client.js

```diff
@@ -446,6 +480,20 @@ const UPSERT_ENTITY_RAW_DATA = gql`
   }
 `;
 
+function getQueryV1Gpl(j1ql, page) {
```

Comment:
This name contains a typo. I'd also prefer dropping the `get` prefix, but this is stylistic; let me know your thoughts!

```suggestion
function queryV1Gql(j1ql, page) {
```

Reply:
Thanks for catching the typo and I agree with you. getQuery isn't really appropriate, maybe something like makeQuery but it's cleanest / clearest in the way you proposed!